packages/utils/src/utils/nullish.utils.ts

```diff
@@ -28,3 +28,12 @@ export const nonNullish = <T>(
  */
 export const notEmptyString = (value: string | undefined | null): boolean =>
   nonNullish(value) && value !== "";
+
+/**
+ * Checks if a given value is null, undefined, or an empty string.
+ *
+ * @param {string | undefined | null} value - The value to check.
+ * @returns {boolean} `true` if the value is null, undefined, or an empty string; otherwise, `false`.
+ */
+export const emptyString = (value: string | undefined | null): boolean =>
```

Comment:
Can we please call it `isEmptyString`? Function names are easier to understand if they start with a verb, and to make it worse `empty` can be interpreted as a verb but it would imply something rather different about what the function does.

Reply:
I think it’s rather consistent to call it `emptyString` since the opposite is named `notEmptyString`. Unless we rename the latter, which I’d prefer to avoid, it’s better to stick with the former in my opinion.

Reply:
We also call it `isNullish` and `nonNullish` where the negative doesn't have the `is`.

Reply:
But if we are fine with a breaking change, not that a strong opinion, we can prefix both with `is` if you rather like?

Reply:
> We also call it isNullish and nonNullish where the negative doesn't have the is.

Good point. Alright.

Reply:
Done. Renamed. You might have to re-approve.